inginious/frontend/templates/task_dispensers/task_list.html

```diff
@@ -1,57 +1,57 @@
 {# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
```

Comment:
There should be no need to change this file as it does not need to be included in the section is hidden.